jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -182,56 +193,27 @@ public void setResources(Resource[] resources)
      *
      * @param csvResources the comma-separated string containing
      * one or more resource strings.
+     * @throws IOException if unable resource declared is not valid
      */
-    public void setResourcesAsCSV(String csvResources)
+    public void setResourcesAsCSV(String csvResources) throws IOException
     {
-        if (csvResources == null)
-        {
-            throw new IllegalArgumentException("CSV String is null");
-        }
-
-        StringTokenizer tokenizer = new StringTokenizer(csvResources, ",;");
-        int len = tokenizer.countTokens();
-        if (len == 0)
+        if (StringUtil.isBlank(csvResources))
         {
-            throw new IllegalArgumentException("ResourceCollection@setResourcesAsCSV(String) " +
-                " argument must be a string containing one or more comma-separated resource strings.");
+            throw new IllegalArgumentException("CSV String is blank");
```

Comment:
This method name is wrong.
This javadoc is wrong.
This message is wrong.

We are not doing CSV, we are doing simple delimited strings (which is not CSV).

Reply:
I don't think this is a big deal ... it's been this way for years without problems, so in the interests of getting 10 out, I'm fine with it as it is.